app/scripts/views/form.js

```diff
@@ -142,25 +148,36 @@ define(function (require, exports, module) {
     },
 
     /**
-     * TODO - this should be called disableSubmit
+     * Disable the form
      */
     disableForm () {
       // the disabled class is used instead of the disabled attribute
       // so that the submit handler is still called. With the submit attribute
       // applied, no submit handler is fired, and the form validation does not
       // take place.
-      this.$('button[type=submit]').addClass('disabled');
-      this._isFormEnabled = false;
+      if (this.isFormEnabled()) {
+        this.$('button[type=submit]').addClass('disabled');
+        this.notifier.trigger('form.disabled');
+      }
     },
 
+    /**
+     * Enable the form
+     */
     enableForm () {
-      this.$('button[type=submit]').removeClass('disabled');
-      this._isFormEnabled = true;
+      if (! this.isFormEnabled()) {
```

Comment:
To avoid sending form.disabled and form.enabled every time either of these methods is called (which is pretty often), I only send the messages if a state change occurs.